lib/runPrettier.js

```diff
@@ -35,7 +35,11 @@ const runPrettier = async ({ write, listDifferent }) => {
     // don't error if `.prettierignore` not found
   }
 
-  prettierArgs.push('**/*.{js,ts,tsx,md,less,css}');
+  if (pathsToCheck.length > 0) {
+    prettierArgs.push(...pathsToCheck);
+  } else {
+    prettierArgs.push('**/*.{js,ts,tsx,md,less,css}');
+  }
```

Comment:
To simplify how this reads, maybe look to default `pathsToCheck` as `['**/*.{js,ts,tsx,md,less,css}']`.